workflow-templates/im-deploy-az-app-service-manually.yml

```diff
@@ -183,11 +167,11 @@ jobs:
 
       # TODO:  If you need to do any Octopus variable substitution (i.e. replacing #{OCTO_PLACEHOLDER} in files) use the following action.  Must be done once per file.  Delete if not using.
```

Comment:
I think we need a new block im-open/set-environment-variables-by-scope block here with a # TODO to fill it out for anything need in appsettings.json replacement or octostatche replacement.  I can't put the suggestion in the right place in the file.  

The octostace/variable-substitution block comments should be updated to indicate they can use env values set in ^^ block within the job.

Reply:
I went ahead and made the changes to the octostache action comment blocks. Let me know if the wording or anything isn't right.

Reply:
I added the new block in the `deploy-code` section of the app service deploy.